nonblock-rpc/rust/core/src/api.rs

```diff
@@ -0,0 +1,13 @@
+use crate::ROUTER;
+
+#[uniffi::export]
+pub fn call(path: String, data: Vec<String>) -> String {
```

Comment:
Nit: I think the term 'path' might be a bit confusing here. Perhaps 'route' would be a better fit.